### PR TITLE
Add example for run VM on separate host (via libvirt_auth)

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

File: examples/bootstraps/centos.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+cd $CREATEPATH
+source bin/activate
+pip install pip --upgrade
+pip install https://github.com/cloudify-cosmo/cloudify-dsl-parser/archive/4.2.zip
+pip install https://github.com/cloudify-cosmo/cloudify-rest-client/archive/4.2.zip
+pip install https://github.com/cloudify-cosmo/cloudify-plugins-common/archive/4.2.zip
+pip install https://github.com/cloudify-cosmo/cloudify-script-plugin/archive/1.5.1.zip
+pip install https://github.com/cloudify-cosmo/cloudify-cli/archive/4.2.zip
+pip install https://github.com/cloudify-cosmo/cloudify-fabric-plugin/archive/1.5.1.zip
+cfy init -r
```
Comment (EarthmanT) [inline examples/bootstraps/centos.sh:26]: `pip install cloudify` should do lines 20-24

File: examples/bootstraps/ubuntu.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+virtualenv centos-libvirt
+cd centos-libvirt
+source bin/activate
+pip install pip --upgrade
+pip install https://github.com/cloudify-cosmo/cloudify-dsl-parser/archive/4.2.zip
+pip install https://github.com/cloudify-cosmo/cloudify-rest-client/archive/4.2.zip
+pip install https://github.com/cloudify-cosmo/cloudify-plugins-common/archive/4.2.zip
+pip install https://github.com/cloudify-cosmo/cloudify-script-plugin/archive/1.5.1.zip
+pip install https://github.com/cloudify-cosmo/cloudify-cli/archive/4.2.zip
+pip install https://github.com/cloudify-cosmo/cloudify-fabric-plugin/archive/1.5.1.zip
```
Comment (EarthmanT) [inline examples/bootstraps/ubuntu.sh:18]: `pip install cloudify` should do lines 13-18

File: examples/cluster/copy_disk.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+{ echo instance-id: iid-local01; echo local-hostname: ${INSTANCE_ID}; } > meta-data
+
+/bin/cat << EOB > user-data
+#cloud-config
+password: passw0rd
+chpasswd: { expire: False }
+ssh_pwauth: True
+sudo: ALL=(ALL) NOPASSWD:ALL
+ssh_authorized_keys:
+  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC50fulmwIfQ2EViqi5qmfCrF8qTxcZTYp4YCwFGi9GptJf8dQK/qiFOjMoL4vo++QfIK4CWWlEc3HyukS8LorVP45syhll7SQ55dG5xGaEGTtCknVep76LjRPWWZIwGEgDmY/Iu8h1Hf00M3bzwOcNfkQ+tPX7hJIfjvMgdKz8eL7ZFzjtnTt02J/uouVpiBTOO6Cb8mIsTnY/Z7HmMJB34h3Fwn+xAVEQY+TDAUJch65XMXD+KUzC52nl5qf+gShYBW1UdmNGCQ41h1u508LHhr+gYSofsk0bRfY5hHuA69qL8MClOjTD2ETowSfdwvOzHgqEJv5ls+gmoQhE8RBwiFP1vbR6XM1NF3FoMe7GnaJAO9grvlYq+XkeBY7JkhiHWosVl9QHRE0gO9e8QYxZH1uT+R5Bmt4oLS1bwQLpxQ4N0GnhRfaDWyhh/KNG/CwoThPwOFzOvm2rCcajiqfgJuU5HVSIfU+Ct4C0J7NYZwzrkZgIcYPTot/69hwiPDQ75Jwowi/19ema0zTHQZUc5fZe+UgK/7lHMllPUbmpkmueSdCQc+fNswtk0k/OCFqFzzU5C20ZB7un44Ledz2HROjozFwNi8uVRBhHrL4ie0FjbuE3i9LybPt9x36vDRNo8H2UDaHDOLF3zSaZjkrseKBKFDaZwCFnxpx5m4ZSaQ== cluster@cloudify.co
```
Comment (EarthmanT) [inline examples/cluster/copy_disk.sh:21]: Probably best if this were a variable and input to operation.

File: examples/cluster/floating_ip.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# add ip alias
+/sbin/ip address add $EXTERNAL_IP/24 dev $EXTERNAL_INTERFACE
```
Comment (EarthmanT) [inline examples/cluster/floating_ip.sh:2]: perhaps a list of commands to execute as part of this phase, but also this is fine.

File: examples/vm_centos.amd64.yaml [hunk 5]
```diff
@@ -0,0 +1,86 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.1/types.yaml
+  - http://www.getcloudify.org/spec/fabric-plugin/1.5.1/plugin.yaml
+  - https://raw.githubusercontent.com/cloudify-incubator/cloudify-utilities-plugin/1.3.1/plugin.yaml
```
Comment (EarthmanT) [inline examples/vm_centos.amd64.yaml:6]: 1.4.4 is latest version.

**Review by EarthmanT — APPROVED** [review]
added comments.

**Comment by 0lvin** [comment]
partially updated in 90f3815b724cf7d358096ff00f1d15c8e335b559

**Pull request merged**